packages/react/src/publication/usePublications.ts

```diff
@@ -15,41 +15,70 @@ import { PaginatedArgs, PaginatedReadResult, usePaginatedReadResult } from '../h
 export type UsePublicationsArgs = PaginatedArgs<PublicationsRequest>;
 
 /**
- * `usePublications` is a paginated hook that lets you fetch publications based on a set of filters.
+ * Fetch a paginated result of publications based on a set of filters.
  *
  * @category Publications
  * @group Hooks
  *
  * @example
- * Fetch publications by a Profile ID
+ * Fetch post publications
  * ```tsx
  * const { data, loading, error } = usePublications({
  *   where: {
- *      from: [profileId('0x0635')]
+ *      publicationTypes: [PublicationType.Post]
+ *    }
+ * });
+ * ```
+ * @example
+ * Fetch comment publications
+ * ```tsx
+ * const { data, loading, error } = usePublications({
+ *   where: {
+ *      publicationTypes: [PublicationType.Comment]
```

Comment:
does it work without profile id at all?

Reply:
sorry, just saw this - but yes it does

Reply:
added these examples to show how people could replicate 

`useComments`
`usePosts`
`useMirrors`
`useQuotes`

with this hook

Reply:
Worth looking at these examples as most likely use case.

If we are to got with `usePublications` for comments (and not implement `useComments`)... which I think it's an interesting idea BTW... maybe a more likely example is "find all comments on a given `publicationId`.